src/pages/documentation/edge-network/validation.md

```diff
@@ -1,27 +1,24 @@
+While the log messages can be quite helpful for an application developer, inspecting the events in the [Assurance UI](https://experience.adobe.com/assurance/) is even easier. In just a few steps, you can view the timestamp ordered list of all the events generated by the Edge Network mobile extension, along with the confirmation that the request was successfully processed by the Edge Network and the enabled server-side solutions.
 
-To use Assurance, follow the [Using Assurance](https://experienceleague.adobe.com/docs/experience-platform/assurance/user-access.html) instructions.
+### Get started with Assurance
 
-### Event types handled by the AEP Edge extension
+1. Start by following the instructions for [using Assurance](https://experienceleague.adobe.com/docs/experience-platform/assurance/user-access.html).
+2. [Install and register the Assurance extension](../platform-assurance-sdk/index.md) in your mobile app.
+3. Follow the same document for [connecting to an Assurance session](../platform-assurance-sdk/index.md#connect-to-an-assurance-session).
 
-Here is a list of the events currently supported by the AEP Edge extension, that can be seen in the Assurance UI:
+### Use the Event Transactions view
 
-* _AEP Request Event_: represents the experience event sent through the AEP Edge extension and contains the XDM and free-form data.
-* _AEP Response Event Handle_: represents the event handle received from Experience Edge in response to an AEP Request Event.
-* _AEP Error Response_: you may see this error event in case of an error with the XDM payload sent to Experience Edge or if a server-side error is returned for other reasons.
-* _service_: this event is published by the Experience Edge Network servers and allows you to validate that the experience event was sent successfully to the services, along with the payload attached to the request, configId used, requestId etc. Examples:
-  * The service events published by `com.adobe.edge.konductor` indicate that a network request was received by the Edge Network.
-  * The service events published by `com.adobe.streaming.validation` indicate if the event passed the XDM format validation based on the schema defined in Adobe Experience Platform. This can help you debug parameter type errors, incorrect parameter names and other errors.
-
-**Hint:** If you want to filter out only the events handled by the AEP Edge extension, you can type one of the event names listed above in the **Search Events** input field from the Session View.
+Inspect and validate the Experience events using the Event Transactions view in Assurance:
 
+1. In the Assurance UI, from the left navigation menu, select **Event Transactions**. If you don't see this option, select **Configure**, then select **Event Transactions** and **Save**.
```

Comment:
nit: I like the context in the experienceleague page more for listing the location of the "Configure" and "add" the Event Transactions.

If you do not see this option, select **Configure** in the bottom left of the window, add the **Event Transactions **, and select **Save**.